### PR TITLE
Added creditCardType function in Finance module

### PR DESCRIPTION
**Pull request by jessleathe**
Added creditCardType function in Finance module and created tests for it as well

### CODE REVIEW

File: src/modules/finance/Finance.cpp [hunk 1]
```diff
@@ -172,4 +172,28 @@ std::string Finance::creditCardExpirationDate()
     const auto expirationDate = Date::futureDate(3);
     return expirationDate.substr(5, 2) + "/" + expirationDate.substr(2, 2);
 }
+
+std::string Finance::creditCardType(const std::string& creditCardNumber)
```
Comment (cieslarmichal) [inline src/modules/finance/Finance.cpp:176]: You did a good job here, but it was about to return just a random credit card type without passing any parameters, please adjust your implementation.

**Review by cieslarmichal — CHANGES_REQUESTED** [review]

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
please change implementation to return random type as string, without passing any number as parameter

**Comment by cieslarmichal** [comment]
hello, are you going to finish this PR? I will close otherwise

File: src/modules/finance/Finance.cpp (hunk 1 above)
Comment (jessleathe) [reply src/modules/finance/Finance.cpp:176]: I am confused on how I fix this. Where should I look to fix it?

Comment (cieslarmichal) [reply src/modules/finance/Finance.cpp:176]: construct a vector with strings: Visa, MasterCard,American Express and Discover and return random element from this vector :) simple as that

Comment (jessleathe) [reply src/modules/finance/Finance.cpp:176]: do I construct the vector in this function or another?

Comment (cieslarmichal) [reply src/modules/finance/Finance.cpp:176]: You can create it in separate file in data directory

Comment (cieslarmichal) [reply src/modules/finance/Finance.cpp:176]: As many others data files

**Pull request closed without merging**